### DISCUSSION TITLE
Contribution planning

### DISCUSSION THREAD

**Discussion by Wobbly21** [razerragin21/Just-Get-Started#5] [General]
Just write here what you're planning on doing so that we aren't working on doing the exact same things to minimise redundancy.
im gonna update license on 9/12
![image](https://user-images.githubusercontent.com/92724702/206325375-2fcb7e89-4c79-4967-ac1c-8844011b9f91.png)
also i'll research what a good license and readme looks like

**Comment by razerragin21** [comment]
Hello all, progess has stagnated, everything okay?

**Comment by razerragin21** [comment]
Let me know if i can invest a few million into employee motivation and democratic leadership.

**Comment by razerragin21** [comment]
@Wobbly21 @Myeser , do let me know if you would like things changed or features implemented, feel free to pull requests on changes and i will review them and push to main accordingly.
